client/src/main/java/io/pravega/client/stream/EventWriterConfig.java

```diff
@@ -26,8 +26,8 @@
 
     public static final class EventWriterConfigBuilder {
         private int initalBackoffMillis = 1;
-        private int maxBackoffMillis = 60000;
```

Comment:
As discussed in the meeting, we may want to be more concervitive here as this is the default and be more aggressive in our tests. IE: 5 or 10s in the tests and 30s here.